### DISCUSSION TITLE
How to cite jaxoplanet?

### DISCUSSION THREAD

**Discussion by zwei-beiner** [exoplanet-dev/jaxoplanet#49] [Q&A] [answered]
I've used jaxoplanet (specifically the ``TransitOrbit`` and ``QuadLightCurve`` classes) and wanted to ask how to cite this? Thank you!

**Comment by dfm** [comment, marked as answer]
@zwei-beiner — Thanks for checking! We don't yet have a citation for jaxoplanet, so perhaps the best bet is to just reference the GitHub repo for now:

```
@software{jaxoplanet,
  author = {Daniel Foreman-Mackey and Lionel J. Garcia},
  title = {{jaxoplanet}: Astronomical time series analysis with {JAX}},
  url = {http://github.com/exoplanet-dev/jaxoplanet},
  version = {0.0.1},
  year = {2023},
}
```
